src/main/java/com/google/sps/utils/StringConstants.java

```diff
@@ -5,31 +5,56 @@
  * throughout the application.
  */
 public final class StringConstants {
-    public static final Double COMMENT_SCORE_THRESHOLD = 0.85;
+  public static final String APPLICATION_NAME = "The Solidarity Initiative";
```

Comment:
Seems like you're organizing the variables by feature.

To make it more readable and to make i easier to find variables, add a comment for each category.

e.g. 
```
/* 
* Calendar constants
*/
```

If there are strings that are used across multiple pages/features, you can have a "common" section too